pom.xml

```diff
@@ -898,6 +898,39 @@
               <exclude>**/interpreter-setting.json</exclude>
               <exclude>**/constants.json</exclude>
               <exclude>scripts/**</exclude>
+              <exclude>**/**/*.log</exclude> 
```

Comment:
Hi @felixcheung ths exclusion already in pom.xml. I just moved by
`f9bc7a9 - Luciano Resende <lresende@apache.org> Sep 5, 2016
[ZEPPELIN-1409] Refactor RAT build on Travis.CI configuration
`
PR

>               <exclude>docs/sitemap.txt</exclude>
              <exclude>docs/search_data.json</exclude>
              <exclude>**/dependency-reduced-pom.xml</exclude>

              <!-- bundled from anchor -->
              <exclude>docs/assets/themes/zeppelin/js/anchor.min.js</exclude>

              <!-- bundled from toc -->